tsconfig.json

```diff
@@ -18,6 +18,20 @@
     "baseUrl": ".",
     "paths": {
       "@/*": ["*"],
+      "@artifacts/*": ["libs/artifacts/*"],
+      "@assets": ["libs/assets"],
+      "@assets/*": ["libs/assets/*"],
+      "@components": ["libs/components"],
+      "@components/*": ["libs/components/*"],
+      "@hooks": ["libs/hooks"],
+      "@hooks/*": ["libs/hooks/*"],
+      "@providers/*": ["libs/providers/*"],
+      "@tokens": ["libs/assets/tokens"],
+      "@tokens/*": ["libs/assets/tokens/*"],
+      "@utils": ["libs/utils"],
+      "@utils/*": ["libs/utils/*"],
+      "@vectors": ["libs/assets/vectors"],
+      "@vectors/*": ["libs/assets/vectors/*"],
       "@polkadot/types": [
```

Comment:
Can't remember if we've decided to switch back to use `@/libs` or have more specific alias like you have here, either way keep them consistent, I see a mixture of `@/libs/` and `@providers`